it/v1/endpoints/ListAllBusinessesControllerISpec.scala

```diff
@@ -129,126 +129,132 @@ class ListAllBusinessesControllerISpec extends IntegrationBaseSpec {
     val desResponseBodyBusinessData: JsValue = Json.parse(
       """
         |{
-        |  "safeId": "XE00001234567890",
```

Comment:
In retrieve we use to have "processingDate": "2023-07-05T09:16:58.655Z" as well, not sure if that is needed here?